keps/sig-storage/1495-volume-populators/README.md

```diff
@@ -271,6 +294,153 @@ downgrade.
 
 No issues
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+_This section must be completed when targeting alpha to a release._
+
+* **How can this feature be enabled / disabled in a live cluster?**
+  - [X] Feature gate (also fill in values in `kep.yaml`)
+    - Feature gate name: AnyVolumeDataSource
+    - Components depending on the feature gate: kube-apiserver
+  - [ ] Other
+    - Describe the mechanism:
+    - Will enabling / disabling the feature require downtime of the control
+      plane?
+    - Will enabling / disabling the feature require downtime or reprovisioning
+      of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
+
+* **Does enabling the feature change any default behavior?**
+  Before this feature, kube API server would silently drop any PVC data source
+  that it didn't recognize, as if the client never populated the field. After
+  enabling this feature, the API server allows any object to be specified.
+  An external admission controller will take over validation of the field,
+  and its response to invalid data sources will be to fail the create operation,
+  rather than silently drop them.
+
+* **Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?**
+  Turning off the feature gate should drop all the data sources that were not
+  already valid and restore the old behavior.
```

Comment:
If a PVC is already created with an external data source, it should continue to work even if you turn off the feature gate afterwards, right?  So restoring the old behavior is only for creating new PVCs.

Reply:
What matter is if the provisioning happened or not already. If something saw the PVC and created the corresponding PV already, then rollback will have no effect on the PV. If the PVC was unbound though then it could lose its data source and any dynamic provisioner would create just an empty volume.

Reply:
Can you add these details to the KEP here?